doc/dynamic-environments/configuration.mkd

```diff
@@ -200,12 +200,81 @@ sources:
 
 ### deploy
 
-The `deploy` setting is a new top level setting for controlling how r10k deploys
+The `deploy` setting is a top level setting for controlling how r10k deploys
 behave. At this point only new settings are included under this setting, but in
 the long term the current top level deploy settings will be moved under
 `deploy`.
 
-#### write_lock
+#### purge\_levels
+
+The `purge_levels` setting controls how aggressively r10k will purge unmanaged
+content during a deployment. Must be a list of values indicating at what levels
+unmanaged content should be purged. Valid values are 'deployment', 'environment',
+and 'puppetfile'.
+
+```yaml
+---
+deploy:
+  purge_levels: [ 'deployment', 'environment', 'puppetfile' ]
+```
+
+This setting currently only impacts the "deploy environment" action.
+
+To maintain parity with existing behavior before this setting was added, the
+default value is `['deployment', 'puppetfile']`.
```

Comment:
How about reverse this sentence?
The default value is `[dep pup]`, which maintains parity...
Much easier to see the default.